_posts/2019-07-10-working-remotely-team.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Tips for Working Remotely in a Team"
+date: 2019-07-02 09:00:00
+categories: ["agile", "remote"]
+author: "emily"
+---
+
+We have been a fully remote company for a year and a half now. Making the transition to being fully remote can be challenging for any team. During the past year and a half we have worked on making sure that our team is as productive, communicative and social as it was when we had a traditional office space. Here are some tips that we have found to be useful for remote teams:
+
+<!--more-->
+
+Have a daily call: We have an office-wide [scrum](https://www.atlassian.com/agile/scrum) call every at 9:45am. We encourage employees to participate via video and to share their goals for the day and any blockers they may have. It is important for teams to have scheduled “face to face” time on a daily basis in order to stay connected. We prefer chatting in the morning.
```

Comment:
@ercohen14 It would be better to link to https://www.atlassian.com/agile/scrum/standups, since that's where it explains about the daily calls (AKA stand-up)